app/src/components/v-form/form-field-interface.vue

```diff
@@ -30,6 +30,13 @@
 			@set-field-value="$emit('setFieldValue', $event)"
 		/>
 
+		<interface-system-raw-editor
+			v-else-if="rawEditorEnabled && rawEditorActive"
+			:value="modelValue === undefined ? field.schema?.default_value : modelValue"
+			:multi-line="field.type === 'text'"
```

Comment:
Should the system interface just auto-switch to multi-line when `type === 'text'` is passed, rather than a new interface option?

```suggestion
			:type="field.type"
```

and then in the interface switch based on the `type` prop 🤔

Reply:
Good call. Definitely more flexible this way! Implemented 👍